### PR TITLE
MBS-9909: Change reports to use React (13 - recordings)

### PR DESCRIPTION
**Pull request by reosarevok**
To be reviewed and merged after https://github.com/metabrainz/musicbrainz-server/pull/931

### CODE REVIEW

**Comment by mwiencek** [comment]
Was it intentional to remove the Artist MBID column from the "Recordings with the same name by different artists with the same name" report? It at least seems somewhat useful where none of the entities have disambiguation comments (so you can see where the artist changes in the list).

File: root/report/RecordingsSameNameDifferentArtistsSameName.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+    <ul>
+      <li>
+        {l(`This report shows all recordings with the same name that have
+            different artists (having different MBIDs) with the same name.`)}
+      </li>
+      <li>
+        {exp.l(`These are most likely cases where the {ac|artist credit} is
+            incorrect for at least one of the recordings.`,
+        {ac: '/doc/Artist_Credits'})}
```
Comment (mwiencek) [inline root/report/RecordingsSameNameDifferentArtistsSameName.js]: Indentation looks a bit wonky on these two lines.

Comment (reosarevok) [reply root/report/RecordingsSameNameDifferentArtistsSameName.js]: Indentation is all fixed in one go in PR 15

**Comment by reosarevok** [comment]
Only one artist doesn't have a disambiguation though (unless something has gone wrong, anyway) since we force entering one after that. So there's no actual benefit to that column.

**Comment by mwiencek** [comment]
Oh, duh. The old page didn't even display the comments.

**Pull request merged**